www/artists/lbeug/ceramics.html

```diff
@@ -20,27 +20,27 @@
                 </p>
                 <div id="gallery">
                     <p>
-                        <a href="http://art.27charisma.com/lbeug/images/albertmemorial.jpg"><img src="images/th_albertmemorial.jpg" alt="no alt text"></a>
+                        <a href="images/th_albertmemorial.jpg"><img src="images/th_albertmemorial.jpg" alt="no alt text"></a>
                         <br>
                         Albert Memorial Bridge. Ceramic, 4 1/4" x 8 1/4"
                     </p>
                     <p>
-                        <a href="http://art.27charisma.com/lbeug/images/bookshelf.jpg"><img src="images/th_bookshelf.jpg" alt="no alt text"></a>
+                        <a href="images/th_bookshelf.jpg"><img src="images/th_bookshelf.jpg" alt="no alt text"></a>
                         <br>
                         Bookshelf III. Ceramic, 8 1/2" x 8 1/4" x 3/4"
                     </p>
                     <p>
-                        <a href="http://art.27charisma.com/lbeug/images/bookshelf4.jpg"><img src="images/th_bookshelf4.jpg" alt="no alt text"></a>
+                        <a href="images/th_bookshelf4.jpg"><img src="images/th_bookshelf4.jpg" alt="no alt text"></a>
                         <br>
                         Bookshelf IV, Ceramic, 8 1/4" x 8 1/4" x 3/4"
                     </p>
                     <p>
-                        <a href="http://art.27charisma.com/lbeug/images/palmtiles.jpg"><img src="images/th_palmtiles.jpg" alt="no alt text"></a>
+                        <a href="himages/th_palmtiles.jpg"><img src="images/th_palmtiles.jpg" alt="no alt text"></a>
                         <br>
                         Palm Tiles. Ceramic
                     </p>
                     <p>
-                        <a href="http://art.27charisma.com/lbeug/images/notebook4.jpg"><img src="images/th_notebook4.jpg" alt="no alt text"></a>
+                        <a href="mages/th_notebook4.jpg"><img src="images/th_notebook4.jpg" alt="no alt text"></a>
```

Comment:
Another typo - "mages" folder instead of "images". Unless you have a folder filled with wizards, then this will break things if you click on this image.